man/systemctl.xml

```diff
@@ -1202,6 +1202,9 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
             be made temporarily in <filename>/run/</filename> and they will be
             lost on the next reboot.</para>
 
+            <para>If <option>--stdin</option> is specified, the new contents will be read from standard
+            input.</para>
```

Comment:
This kinda stretches the concept of "editing". it doesn#t edit after all, it just replaces. editing (at least to me) means changing things so that part of the old and part of the new are in the end result.

I think this at least deserves some mention in the docs here, i.e. "In this mode the old contents of the unit file is fully replaced by the new provided definition." or so.

I think it's OK to stretch the word "edit" like this, I just want that this is made explicit that the old version of the file does not play any role anymore at all.